### PR TITLE
set convention for repo layout

### PR DESCRIPTION
**Pull request by ohlsont**

### CODE REVIEW

File: go/README.md [hunk 1]
```diff
@@ -41,8 +41,7 @@ optimize for brevity.
 
 ## Directory layout
 
-**TODO:** Decide if to adopt [Ben Johnson's standard package
-layout](https://medium.com/@benbjohnson/standard-package-layout-7cdbc8391fc1).
+Follow [The Golang standards repos lyout](https://github.com/golang-standards/project-layout)
```
Comment (odsod) [inline go/README.md]: Spelling: lyout

Comment (odsod) [reply go/README.md]: Add trailing period, since this is a full sencence.

Comment (odsod) [reply go/README.md]: Use the same title text for the link as the repo title: `[Standard Go Project Layout](...)`

**Comment by odsod** [comment]
Please capitalize the commit message heading

**Comment by Jassob** [comment]
Looks good now :+1:

**Comment by odsod** [comment]
:+1:

**Pull request merged**